sample.env

```diff
@@ -1,305 +1,421 @@
+GOOGLE_CLIENT_ID=
+GOOGLE_CLIENT_SECRET=
+
+#### ---- cvmanager_postgres
+
+# PostgreSQL Database connection information
+# this value may need to follow with the webapp host if debugging the applications 
+PG_DB_HOST=${DOCKER_HOST_IP}:5432
+PG_DB_NAME=postgres
+PG_DB_USER=postgres
+# If the PG_DB_PASS variable has special characters, make sure to wrap it in single quotes
+PG_DB_PASS=postgres
+
+#### ---- cvmanager_api
+
+# Allowed CORS domain for accessing the CV Manager API from (set to the web application hostname)
+# Make sure to include http:// or https://
+# If using docker then this value should be set to: http://${WEBAPP_HOST_IP}:3000
+# If running the webapp using npm then set it to: http://localhost:3000
+# Leave as * to allow all domains access
+CORS_DOMAIN=*
+
+# Set these variables if using either "MONGODB" or "BIGQUERY" for COUNT_DESTINATION_DB of jpo_count_metric
+# COUNTS_MSG_TYPES: Comma separated list of message types.
+# COUNTS_MSG_TYPES must be set for the counts menu to correctly populate when building an image for deployment
+COUNTS_MSG_TYPES='BSM,SSM,SPAT,SRM,MAP'
+GEO_DB_NAME='V2XGeoJson'
+SSM_DB_NAME=
+SRM_DB_NAME=
+
+# If jpo_count_metric COUNT_DESTINATION_DB is "MONGODB", set the MongoDB variables
+MONGO_DB_URI=
+MONGO_DB_NAME="ode"
+
+# Specifies the maximum number of V2x messages returned from the geo_query_geo_data_mongo method before filtering occurs
+MAX_GEO_QUERY_RECORDS=
+
+# If running firmware manager addon
+FIRMWARE_MANAGER_ENDPOINT=http://${DOCKER_HOST_IP}:8089
+
+# If connecting to PGDB over websocket:
+INSTANCE_CONNECTION_NAME=
+
+# Python timezone for the CV Manager (You can list pytz timezones with the command 'pytz.all_timezones')
+TIMEZONE="US/Mountain"
+
+# WZDx API key and endpoint for pulling WZDx data into the CV Manager
+WZDX_API_KEY=
+WZDX_ENDPOINT=data.cotrip.org
+
+# Contact Support Menu Email Configuration
+CSM_EMAIL_TO_SEND_FROM=
+CSM_EMAIL_APP_USERNAME=
+CSM_EMAIL_APP_PASSWORD=
+CSM_EMAILS_TO_SEND_TO=
+CSM_TARGET_SMTP_SERVER_ADDRESS=
+CSM_TARGET_SMTP_SERVER_PORT=587
+CSM_TLS_ENABLED=true
+CSM_AUTH_ENABLED=true
+CSM_EMAIL_APP_USERNAME=
+CSM_EMAIL_APP_PASSWORD=
+
+
+######## -------- "webapp" Docker Profile Services
+# Run webapp service for cvmanager
+# Requires: basic
+# Compose file: docker-compose.yml
+# cvmanager_webapp
+# - React frontend for cvmanager
+
+# Mapbox token for map rendering in the webapp
+MAPBOX_TOKEN=
+
+# DOT_NAME must be set for the DOT name to correctly populate when building an image for deployment
+DOT_NAME="CDOT"
+
+# Initial map viewport
+MAPBOX_INIT_LATITUDE="39.7392"
+MAPBOX_INIT_LONGITUDE="-104.9903"
+MAPBOX_INIT_ZOOM="10"
+
+VIEWER_MSG_TYPES='BSM'
+
+CVIZ_API_SERVER_URL=http://${DOCKER_HOST_IP}:8089
+CVIZ_API_WS_URL=ws://${DOCKER_HOST_IP}:8089
+
+# Webapp themes: light, dark, cdotDark
+# base theme is used by default, dark theme is used if browser is set to dark mode
+WEBAPP_THEME_LIGHT="cdotDark" # if not set, defaults to 'light'
```

Comment:
I wouldn't be against setting this to just dark in the sample since we have pipelines in place to inject the proper environment variables for CDOT. We just need to make sure those pipelines get updated. Although I suppose we do have CDOT as the default for other variables. I am fine either way.